client/my-sites/sidebar/sidebar.jsx

```diff
@@ -265,7 +265,8 @@ module.exports = React.createClass( {
 			return null;
 		}
 
-		if ( ! config.isEnabled( 'manage/plugins/wpcom' ) && ! this.props.sites.hasSiteWithPlugins() ) {
+		if ( ! this.props.sites.hasSiteWithPlugins() &&
+			( ! config.isEnabled( 'manage/plugins/wpcom' ) || abtest( 'wpcomPluginsInSidebar' ) === 'hidePlugins' ) ) {
```

Comment:
I think the check should continue if the config.isEnabled check is `true` which is why I used an `||`. Maybe it'd be clearer to separate these onto their own lines?

```
if ( ! config.isEnabled( 'manage/plugins/wpcom' ) {
    return;
}

if ( abtest( 'wpcomPluginsInSidebar' === 'hidePlugins' ) {
    return;
}
```